### ISSUE TITLE
Issues in Directory Opus

### ISSUE THREAD

**Issue by vjalle** [be5invis/Iosevka#3154]
Hi,

This font does not work properly in Directory Opus 13.22:
- If it's set as the "File display" font, performance drops to unacceptable level. Resizing lister windows becomes very slow, keyboard navigation cannot follow the key repetition rate anymore and gets slow. The bigger the window, the worse is the issue: https://resource.dopus.com/t/cpu-load-during-scrolling-resizing-listers-etc/59095/10
- If it's set as "Code editors" font, there are graphical artifacts: https://resource.dopus.com/t/text-dimensions-bad-cursor-position-incorrect-in-the-hotkey-editor/59115/10

See the issue reports with images in the DOpus forums, links above.

---

Custom build of 34.3 (latest), no change to metrics. Inherited SS20 Curly, see the toml file in the DOpus forum. Only changes some chars and disables ligations.
Windows 11 (latest).

Hint from Leo Davidson (DOpus dev):
"Tell the font author there’s a problem with strings that start with a zero-width space. That may be enough for them to work out what’s going wrong in the font’s code or metrics."

Best regards, Ferenc
